docs/courses/csintro1/motion/create-on-create-sprites.md

```diff
@@ -146,14 +147,14 @@ cloud = sprites.create(img`
 
 ## Student Task #1: More Random Clouds
 
-The ``||sprites:on created||`` event allows us to set code to run whenever a new sprite is created. This is used to create new clouds multiple times with the same code. Now we will create new clouds with ``||sprites:sprite of kind||`` blocks.
+The ``||sprites:on created||`` event allows us to set code to run whenever a new sprite is created. This is used to create new clouds multiple times with the same code. Now we will create new clouds with ``||variables:set cloud to||`` blocks with empty image editors.
 
 1. Start with example #1 or your own similar code
-2. Add 2 more ``||sprites:sprite of kind||`` blocks for clouds
-3. Add a new ``||sprites:sprite of kind||`` for a different ``||sprites:kind||``. Use the ``||sprites:on created||`` event to
+2.	Add two more ``||variables:set cloud to||`` blocks for clouds
```

Comment:
```suggestion
2. Add two more ``||variables:set cloud to||`` blocks for clouds
```

These tabs don't show up, do they? I think leading spaces / tabs get trimmed so they just show up like other lists; if so preferably keep these as just spaces so that it doesn't look like they're intended to be indented.